### PR TITLE
feat: Cleanup (PR-3)

### PR DESCRIPTION
**Pull request by iamsivin**
# Pull Request Template

## Description

This PR includes removing SCSS files and moving them to component and inline.

### CODE REVIEW

File: app/javascript/dashboard/assets/scss/_utility-helpers.scss [hunk 1]
```diff
@@ -1,79 +1 @@
-.margin-bottom-small {
```
Comment (iamsivin) [inline app/javascript/dashboard/assets/scss/_utility-helpers.scss:1]: I moved all these class names to inline tailwind.

File: app/javascript/dashboard/assets/scss/widgets/_conversation-card.scss [hunk 2]
```diff
@@ -1,16 +1 @@
-@keyframes left-shift-animation {
```
Comment (iamsivin) [inline app/javascript/dashboard/assets/scss/widgets/_conversation-card.scss:1]: I moved this to the helper classes SCSS file.

File: app/javascript/dashboard/assets/scss/widgets/_conversation-view.scss [hunk 3]
```diff
@@ -1,249 +1 @@
-// scss-lint:disable MergeableSelector
```
Comment (iamsivin) [inline app/javascript/dashboard/assets/scss/widgets/_conversation-view.scss:1]: I moved all the styles to `messageView` component without scoping.

File: app/javascript/dashboard/assets/scss/widgets/_search-box.scss [hunk 4]
```diff
@@ -1,18 +1 @@
-.search {
```
Comment (iamsivin) [inline app/javascript/dashboard/assets/scss/widgets/_search-box.scss:1]: We are not using this class name anywhere. I think it's related to our old search.

File: app/javascript/dashboard/assets/scss/widgets/_sidemenu.scss [hunk 5]
```diff
@@ -1,78 +1 @@
-.side-menu {
```
Comment (iamsivin) [inline app/javascript/dashboard/assets/scss/widgets/_sidemenu.scss:1]: We are not using these class names anywhere.

**Review by scmmishra — APPROVED** [review]

**Pull request merged**